From e13db0f16c4ac5a243587e881d7b18979d07f478 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Caol=C3=A1n=20McNamara?= <caolanm@redhat.com>
Date: Mon, 25 Jul 2016 12:08:35 +0100
Subject: curl 7.50.0 has CURL as typedef struct Curl_easy

Change-Id: I22e5e2cdf78c38087579071c1b1570a8adc7d3c4
(cherry picked from commit 0b8e589875ffd84150470832de18ebd79989efc0)
Reviewed-on: https://gerrit.libreoffice.org/27513
Reviewed-by: David Tardon <dtardon@redhat.com>
Tested-by: Jenkins <ci@libreoffice.org>

diff --git a/ucb/source/ucp/ftp/ftploaderthread.cxx b/ucb/source/ucp/ftp/ftploaderthread.cxx
index 25bc26d..5d1cfb5 100644
--- a/ucb/source/ucp/ftp/ftploaderthread.cxx
+++ b/ucb/source/ucp/ftp/ftploaderthread.cxx
@@ -77,7 +77,7 @@ FTPLoaderThread::~FTPLoaderThread() {
 
 
 CURL* FTPLoaderThread::handle() {
-    CURL* ret = osl_getThreadKeyData(m_threadKey);
+    CURL* ret = static_cast<CURL*>(osl_getThreadKeyData(m_threadKey));
     if(!ret) {
         ret = curl_easy_init();
         if (ret != nullptr) {
-- 
cgit v0.10.2

